x-pack/plugins/monitoring/server/lib/alerts/fetch_index_shard_size.test.ts

```diff
@@ -187,6 +188,7 @@ describe('fetchIndexShardSize', () => {
                           '_index',
                           'index_stats.shards.primaries',
```

Comment:
should the ecs equivalent of `index_stats.shards.primaries` be  `elasticsearch.index.shards.total` or `elasticsearch.cluster.stats.indices.shards.primaries`?

Reply:
So the property should be `elasticsearch.index.shards.primaries` but it looks like it is [omitted when serialized to json](https://github.com/elastic/beats/blob/main/metricbeat/module/elasticsearch/index/data.go#L116-L128) in the 8.x metricbeat version. Looking at the < 8 version, these [properties are serialized as expected](https://github.com/elastic/beats/blob/7.16/metricbeat/module/elasticsearch/index/data_xpack.go#L108-L120)

Reply:
I rebuilt metricbeat replacing the `-` with expected property names and they're now surfacing. These properties are not mapped, do we need them to be mapped for this query to work ?

<img width="482" alt="Screenshot 2022-02-15 at 18 30 12" src="https://user-images.githubusercontent.com/5239883/154116526-2d1d610b-abf8-4fd7-92a4-7be6ce085179.png">

Reply:
Thanks Kevin!  No, I don't think it must be mapped as its referenced in `_source`.  I'll add the path.